src/project-2.js

```diff
@@ -44,62 +72,84 @@ const isPrime = (num) => {
   // hint: a prime number is only evenly divisible by itself and 1
   // hint2: you can solve this using a for loop
   // note: 0 and 1 are NOT considered prime numbers
+  if (num === 0) {
+    return false;
+  }
+  for (let i = 2; i < num; i++) {
```

Comment:
Good job.  You could also skip all of the even numbers to make it more efficient.  You can also just iterate up to the square root of the number.  Check out the Sieve of Eratosthenes on Wikipedia